draft-ietf-mops-streaming-opcons.md

```diff
@@ -341,6 +371,60 @@ We note that other operators saw similar spikes during this time period. Craig L
+  * non-low-latency live (10 seconds to a few minutes)
+  * on-demand            (hours or more)
+
+## Ultra Low Latency
+
+Ultra low latency delivery of media is defined here as having a glass-to-glass delay target under one second.
+
+This level of latency is sometimes necessary for real-time interactive applications such as video conferencing, operation of remote control devices or vehicles, or remotely hosted real-time gaming systems.  Some media content providers aim to achieve this level of latency for live media events involving sports, but have usually so far been unsuccessful over the internet at scale, though it is often possible within a localized environment with a controlled network, such as inside a specific venue connected to the event.  Applications operating in this domain that encounter transient network events such as loss or reordering of some packets often experience user-visible artifacts in the media.
+
+Applications requiring ultra low latency for media delivery are usually tightly constrained on the available choices for media transport technologies, and sometimes may need to operate in controlled environments to reliably achieve their latency and quality goals.
+
+Most applications operating over IP networks and requiring latency this low use the real-time transport protocol (RTP) {{RFC3550}} or WebRTC {{I-D.draft-ietf-rtcweb-overview-19}}{{WEBRTC}}, which uses RTP for the media transport as well as several other protocols necessary for safe operation in browsers.
+
+Worth noting is that many applications for ultra low latency delivery do not need to scale to more than one user at a time, which simplifies many delivery considerations relative to other use cases.  For applications that need to replicate streams to multiple users, especially at a scale exceeding tens of users, this level of latency has historically been nearly impossible to achieve except with the use of multicast or planned provisioning in controlled networks.
+
+Recommended reading for applications adopting an RTP-based approach also includes {{RFC7656}}.  Applications with further-specialized latency requirements are out of scope for this document, but {{TS102929}} and {{TS26448}} may be helpful as further reading for operators who need to consider attempting to achieve latency at this scale.
+
+## Low Latency Live
+
+Low latency live delivery of media is defined here as having a glass-to-glass delay target under 10 seconds.
+
+This level of latency is targeted to have a user experience similar to traditional broadcast TV delivery.  A frequently cited problem with failing to achieve this level of latency for live sporting events is the user experience failure from having crowds within earshot of one another who react audibly to an important play, or from users who learn of an event in the match via some other channel, for example a messaging application such as Twitter, before it has happened on the screen showing the sporting event.
```

Comment:
For Twitch, our primary driving factor has been to reduce latency for the sake of chat. A user can leave a comment on a live stream and the broadcaster will read it in real-time. However, latency causes the broadcaster's verbal response to be delayed.

It might be worth generalizing this as any form of social chatter. This includes being able to influence the broadcast itself.

Reply:
> It might be worth generalizing this as any form of social chatter. This includes being able to influence the broadcast itself.

Maybe the short list of use cases at the very top of https://github.com/Dash-Industry-Forum/dash.js/wiki/Low-Latency-streaming might help?

Reply:
I think this is covered adequately in the new PR.